staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -233,6 +234,9 @@ type Config struct {
 
 	// StorageVersionManager holds the storage versions of the API resources installed by this server.
 	StorageVersionManager storageversion.Manager
+
+	// RequestWidthFinder is used to find the "width" of the incoming request(s).
+	RequestWidthFinder width.RequestWidthFinder
```

Comment:
RequestWidthEstimator ? [I think this is my personal preference, but please pushback if you disagree]

RequestWithCalculator ?

Reply:
i originally had `RequestWithCalculator`, but i like `RequestWidthEstimator` better :)